### DISCUSSION TITLE
I can't reach my database

### DISCUSSION THREAD

**Discussion by serkan-bayram** [vercel/next.js#57719] [Help]
### Summary

I have deployed my project to vercel and I'm using PocketHost to host my database, PocketHost gives you an url to make your api calls and it works fine on local, but I couldn't make it work on vercel deployment, I think it has something to do with env variables but I don't know, anybody have an idea?

**Comment by icyJoseph** [comment]
What errors do you actually see?

Have you set API keys and others in the Vercel Dashboard?
